src/frontend/sharedTypes/navigation.ts

```diff
@@ -43,7 +43,16 @@ export type RootStackParamsList = {
   CoordinateFormat: undefined;
   Experiments: undefined;
   PhotoPreviewModal: {
+    createdByDeviceId?: string;
```

Comment:
I opted to make this a nav param as opposed to something that this screen fetches internally, with the intention that this screen does as little work as possible when it comes to fetching data.

Open to changing but I think it's generally worked out okay so far 😄

Reply:
I suspect that the photo metadata will live on the `photo` nav prop, so no additional nav params should be needed to access those when they're introduced.